output.go

```diff
@@ -25,17 +25,21 @@ import (
 	"io"
 	"log"
 	"os"
+
+	"github.com/uber-go/zap"
 )
 
 type output interface {
 	io.Writer
+	zap.Logger
 
 	Fatalf(format string, args ...interface{})
 	Printf(format string, args ...interface{})
 }
 
 type consoleOutput struct {
 	*os.File
+	zap.Logger
```

Comment:
it is now somewhat odd that `consoleOutput.Printf` and `log.Fatalf` do not use the `zap.Logger` available